### PR TITLE
preact support

### PR DESCRIPTION
**Pull request by Merciful12**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +74,7 @@
     "jest-localstorage-mock": "^2.4.12",
     "lint-staged": "^11.0.0",
     "nanodelay": "^2.0.0",
+    "preact": "^10.5.13",
```
Comment (ai) [inline package.json:77]: Do we need to add optional peer dependency as we did for `react`?

**Comment by ai** [comment]
Also, what is your Twitter&Telegram accounts to mention you in releae tweet?

File: package.json (hunk 1 above)
Comment (Merciful12) [reply package.json:77]: I think yes, but not sure which version it should be

**Comment by Merciful12** [comment]
> Also, what is your Twitter&Telegram accounts to mention you in release tweet?

Twitter: `@12Future12`
Telegram: `@Merciful`

**Comment by ai** [comment]
Is it OK if I will release it a few days later? You do not pan to use it soon, don't you?

**Comment by ai** [comment]
To have full Preact support we need to port this fles too https://github.com/logux/client/tree/main/react

**Pull request merged**